dds/idl/be_produce.cpp

```diff
@@ -206,6 +206,7 @@ void postprocess(const char* fn, ostringstream& content,
     }
     if (which == BE_GlobalData::STREAM_H) {
       out << "#include \"dds/DCPS/Definitions.h\"\n";
+      out << "#include \"dds/DdsDcpsC.h\"\n";
```

Comment:
We need DdsDcpsC.h else we don't get all IDL defined DDS types visible for the user. Tried some more specific but that lead to compile issues in tests